frontend/src/App.js

```diff
@@ -0,0 +1,44 @@
+import './App.css';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Navbar from './components/navbar.jsx';
+import Home from './components/home.jsx';
+import NewRecipe from './components/newRecipe.jsx';
+import About from './components/about.jsx';
+import Recipe from './components/recipe.jsx';
+import { useState, useEffect } from 'react';
+
+function App() {
+	const [recipes, setRecipes] = useState([]);
+
+	useEffect(() => {
+		fetch('/api/recipe')
+			.then((res) => res.json())
+			.then((data) => setRecipes(data))
+			.catch((err) => console.log(err));
+	}, []);
+
+	return (
+		<Router>
+			<div className='App'>
+				<Navbar />
+				<Switch>
+					<Route exact path='/' component={Home} />
+					<Route path='/new' component={NewRecipe} />
+					<Route path='/about' component={About} />
+					{recipes.map((recipe) => {
+						return (
+							<Route path={`/recipes/${recipe.name}`}>
```

Comment:
FYI your code currently has a bug that prevents me from opening your third recipe, Hot Dry Noodles (Re Gan Mian). After a bit of experimenting, this appears to be because of the parenthesis in the name of the dish, which are messing up the url. To fix, use a node package to generate url slugs or just remove parenthesis from all urls.